CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ set(GLAD_API "" CACHE STRING "API type/version pairs, like \"gl=3.2,gles=\", no
 set(GLAD_GENERATOR "c" CACHE STRING "Language to generate the binding for")
 set(GLAD_EXTENSIONS "" CACHE STRING "Path to extensions file or comma separated list of extensions, if missing all extensions are included")
 set(GLAD_SPEC "gl" CACHE STRING "Name of the spec")
+option(GLAD_ALL_EXTENSIONS "Include all extensions iso those specified by GLAD_EXTENSIONS" OFF)
```

Comment:
Typo in the description?

Reply:
Idiosyncrasy of mine: iso = instead of. Will adjust